_posts/2017-06-27-Increasing-Rusts-Reach.md

```diff
@@ -0,0 +1,310 @@
+---
+layout: post
+title: "Increasing Rust’s Reach"
+author: Carol Nichols
+---
+
+One of [Rust’s 2017 goals](https://blog.rust-lang.org/2017/02/06/roadmap.html) is that Rust should
+have a lower learning curve and be easier to become productive in. We want Rust to be accessible to
```

Comment:
Nit: I'm not normally persnickety about this sort of thing, but the dangling participle in "and be easier to become productive in" is really throwing me. Perhaps something like this?

One of our 2017 goals is to make it easier to become productive in Rust by reducing its learning curve.